composer.lock

```diff
@@ -8,16 +8,16 @@
     "packages": [
         {
             "name": "async-aws/core",
-            "version": "1.22.1",
+            "version": "1.26.0",
```

Comment:
Are all composer lock changes needed?

Reply:
Yep, but they are not related to the current topic, it came from master.